serving/src/main/java/feast/serving/service/RedisServingService.java

```diff
@@ -261,7 +261,7 @@ private void sendAndProcessMultiGet(
             .forEach(
                 f -> {
```

Comment:
Can you please write out the whole variable name. It's quite confusing if its just `f` since `f` is a FeatureRow but `fr` is a FeatureReference, when in fact `fr` is actually a FeatureSetReference.